docs-source/content/userguide/managing-domains/domain-events.md

```diff
@@ -34,11 +34,16 @@ The operator generates these event types in a domain namespace, which indicate t
  *  `NamespaceWatchingStarted`: The operator has started watching for domains in a namespace.
  *  `NamespaceWatchingStopped`: The operator has stopped watching for domains in a namespace. Note that the creation of this event in a domain namespace is the operator's best effort only; the event will not be generated if the required Kubernetes privilege is removed when a namespace is no longer managed by the operator.
 
+The operator also generates these event types in the operator's namespace, which indicate the following:
+
+*  `StartManagingNamespace`: The operator has started managing domains in a namespace.
```

Comment:
Is there a reason why we cannot also use these new event types in domain namespaces to replace NamespaceWatchingStarted and NamespaceWatchingStopped?

Reply:
It would definitely be better if we could have less event types. I needed to make the events type different in domain namespace and operator's namespace mainly because in the dedicated mode, we don't want to see two set of events with the same type. We need to keep the events in both domain namespace and operator namespace because the administrator who manages a domain namespace may not have access to the operator's namespace.

Reply:
Another reason that we use different event types are that the involvedObject points to different resources, NamespaceWatchingStarted/Stopped involves the namespace resource, and Start/StopManagingNamespaces involve the operator pod resource.